### ISSUE TITLE
Panic runtime error while using UDP ingress route

### ISSUE THREAD

**Issue by alekitto** [containous/traefik#6494]
### Do you want to request a *feature* or report a *bug*?

Bug

### What did you do?

Hi, i'm using traefik to route UDP packets to jitsi videobridge deployed on kubernetes.
After a couple of minutes, the traefik pod crashes with `panic: runtime error: racy use of timers`

### What did you expect to see?

No error.

### What did you see instead?

```
panic: runtime error: racy use of timers

goroutine 605 [running]:
time.resetTimer(0xc000416c38, 0xdb58335d04d2)
	/usr/local/go/src/runtime/time.go:233 +0x35
time.(*Timer).Reset(0xc000416c30, 0xb2d05e00, 0x2)
	/usr/local/go/src/time/sleep.go:126 +0x7d
github.com/containous/traefik/v2/pkg/udp.(*Conn).Read(0xc0005a85b0, 0xc0062a0000, 0x8000, 0x8000, 0x41, 0x0, 0x0)
	/go/src/github.com/containous/traefik/pkg/udp/conn.go:233 +0x176
io.copyBuffer(0x7fe95c435fb8, 0xc00000e0e8, 0x2c22b00, 0xc0005a85b0, 0xc0062a0000, 0x8000, 0x8000, 0x456ddb8, 0x2c89ae0, 0xc00222af00)
	/usr/local/go/src/io/io.go:405 +0x122
io.Copy(...)
	/usr/local/go/src/io/io.go:364
github.com/containous/traefik/v2/pkg/udp.Proxy.connCopy(0xc0009a01b0, 0x10, 0x2c502a0, 0xc00000e0e8, 0x2c22b00, 0xc0005a85b0, 0xc000293620)
	/go/src/github.com/containous/traefik/pkg/udp/proxy.go:50 +0xa1
created by github.com/containous/traefik/v2/pkg/udp.(*Proxy).ServeUDP
	/go/src/github.com/containous/traefik/pkg/udp/proxy.go:39 +0x3ec
```

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
Version:      2.2.0-rc2
Codename:     chevrotin
Go version:   go1.14
Built:        2020-03-11T17:42:16Z
OS/Arch:      linux/amd64
```

**Comment by ldez** [comment]
Hello,

could you provide a reproducible example?

**Comment by traefiker** [comment]
Closed by #6498.
